### PR TITLE
Add timestamp to some superpmi.py output

### PR DESCRIPTION
**Pull request by BruceForstall**
This should help determine where time is being spent in long-running
tasks such as PMI or crossgen2 collections in the CI system.

### CODE REVIEW

**Comment by BruceForstall** [comment]
@kunalspathak PTAL
cc @dotnet/jit-contrib

File: src/coreclr/scripts/superpmi.py [hunk 1]
```diff
@@ -402,15 +408,15 @@ def write_file_to_log(filepath, log_level=logging.DEBUG):
     if not os.path.exists(filepath):
         return
 
-    logging.log(log_level, "============== Contents of %s", filepath)
+    logging.log(log_level, "%s============== Contents of %s", get_time_prefix(), filepath)
```
Comment (kunalspathak) [inline src/coreclr/scripts/superpmi.py:411]: Don't think printing time here is of any use. The real time prefix needs to happen at the point where the contents in the `filepath` were created.

**Review by kunalspathak — CHANGES_REQUESTED** [review]
I thought we would have a timeprefix before the line that `superpmi.exe` prints, to understand if particular `.dll` is taking time or not? In latest runs, we see that crossgen2 win-arm64 timed out , but we need to see and compare which `.dll(s)` are more time consuming compared to win-x64 counter parts. The time prefix at beginning/end would still help, but not sure how useful it would be.

File: src/coreclr/scripts/superpmi.py (hunk 1 above)
Comment (BruceForstall) [reply src/coreclr/scripts/superpmi.py:411]: It's true it might not be what we might want to see, but it does provide at least some information.

**Comment by BruceForstall** [comment]
The main potential benefit here is the line that prints out status, e.g., `[1:23]: `, etc., by prefixing a time. So we would see when each dll begins being processed.

**Comment by kunalspathak** [comment]
Sure, lets give it a try and see if it helps.

**Review by kunalspathak — APPROVED** [review]

**Pull request merged**